assets/js/modules/search-console/components/dashboard-details/DashboardDetailsWidgetKeywordsTable.js

```diff
@@ -32,18 +32,23 @@ import Layout from '../../../../components/layout/layout';
 import { STORE_NAME } from '../../datastore/constants';
 import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
 import { getCurrentDateRangeDayCount } from '../../../../util/date-range';
+import { unTrailingSlashIt } from '../../../../util';
 
 const { useSelect } = Data;
 
 const DashboardDetailsWidgetKeywordsTable = () => {
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
 	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const isDomainProperty = useSelect( ( select ) => select( STORE_NAME ).isDomainProperty() );
+	const referenceSiteURL = useSelect( ( select ) => unTrailingSlashIt( select( CORE_SITE ).getReferenceSiteURL() ) );
 	const footerCTALinkArgs = {
 		resource_id: propertyID,
 		num_of_days: getCurrentDateRangeDayCount(),
 	};
 	if ( url ) {
```

Comment:
The `getCurrentEntityURL()` selector [will use `getReferenceSiteURL()` as a fallback](https://github.com/google/site-kit-wp/blob/5e51067e3a4bc93f7cd5419d798f6ee2d0f56713/assets/js/googlesitekit/datastore/site/info.js#L351-L353), so I don't see how this check could ever be `false` but have the `else if` be `true`.

Reply:
@tofumatt That's only happening in the `getCurrentReferenceURL` selector, but not the `getCurrentEntityURL` selector which is being used here.